Python/getversion.c

```diff
@@ -13,3 +13,6 @@ Py_GetVersion(void)
                   PY_VERSION, Py_GetBuildInfo(), Py_GetCompiler());
     return version;
 }
+
+/* Export version as API data */
```

Comment:
I don't get what you mean by "API data". Maybe:

```suggestion
// Export the Python version as a constant variable.
```

Reply:
I worded this based on the actual definition:

~~~
PyAPI_DATA(const long) Py_Version;
~~~

as I thought this conveyed what the new symbol actually is. I'll reword this.